deployment/jobservice/templates/statefulset.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ include "jobservice.name" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "jobservice.labels.all" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicas }}
```

Comment:
If we only support 1 replica, should replicas be configurable?

Reply:
Done.